serf/event.go

```diff
@@ -156,13 +158,53 @@ func (q *Query) Respond(buf []byte) error {
 		return fmt.Errorf("response exceeds limit of %d bytes", q.serf.config.QueryResponseSizeLimit)
 	}
 
-	// Send the response
```

Comment:
I think we still want to keep the direct response as well. This'll keep things working if there are no other members in the cluster that support the relay.